docs/README.md

```diff
@@ -1 +1,29 @@
 # Welcome to Quarantine Fourteen
+
+# Introduction
+
+You're stuck in your apartment for a 14-day quarantine - how do you survive? Quarantine Fourteen is a text-based adventure game simulating the conditions that many of us are in right now.
+
+# Gameplay
+
+Your character has 2 key stats: energy and fulfillment. What's the goal? Make it out of quarantine with the maximum energy and fulfillment!
+
+10 times a day, you pick an action that will change your stats in different ways. Choose wisely! The actions you choose could have different consequences depending on how they're combined.
```

Comment:
nitpick: consequences -> effects? Consequences has a kinda negative connotation

Reply:
good point! I'll amend it